docs/counter.md

```diff
@@ -114,7 +110,7 @@ If you see the source files instead of the example, click **View App** in the bo
 
 <div class="glitch-embed-wrap" style={{ height: '420px', width: '100%' }}>
   <iframe
-    src="https://glitch.com/embed/#!/embed/wm-count-revenue?path=README.md&previewSize=100"
+    src="https://glitch.com/embed/#!/embed/wm2-count-revenue?path=README.md&previewSize=100"
```

Comment:
I think the "Thanks to you, I have made nothing (yet)" in the glitch a bit passive aggressive.